recipes/libxml2/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+if [[ `uname` == 'Darwin' ]];
+then
+    export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+elif [[ `uname` == 'Linux' ]];
+then
+    export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+fi
+
+./autogen.sh
+./configure --prefix="${PREFIX}" \
+            --with-iconv="${PREFIX}" \
+            --with-icu \
```

Comment:
So, `./configure` doesn't say a path can be provided here. However, I tried to do this. It seemed to run through. Though the 10 test failures returned. Very strange.